thoth/common/openshift.py

```diff
@@ -1479,6 +1481,7 @@ def schedule_adviser(
         github_installation_id: Optional[int] = None,
         github_base_repo_url: Optional[str] = None,
         re_run_adviser_id: Optional[str] = None,
+        source_type: Optional[ThothAdviserIntegrationEnum] = None,
```

Comment:
We can remove `is_s2i` here and below

Reply:
I can add a TODO so @fridex can take it, I don't know how it can affect the thoth-s2i image if we remove that parameter right now. ack?

Reply:
ack. Good call :+1: